service/controller/app/resource/configmap/resource.go

```diff
@@ -7,12 +7,17 @@ import (
 	"github.com/giantswarm/micrologger"
 	corev1 "k8s.io/api/core/v1"
 
+	"github.com/giantswarm/app-operator/service/controller/app/controllercontext"
 	"github.com/giantswarm/app-operator/service/controller/app/values"
 )
 
 const (
 	// Name is the identifier of the resource.
 	Name = "configmap"
+
+	// configmapMergeFailedStatus is set in the CR status when there is an failure during
+	// merge configmaps.
+	configmapMergeFailedStatus = "configmap-merge-failed"
```

Comment:
See my other comment. I think it would be better to make this public and only check for these statuses. WDYT?